zulipterminal/ui_tools/views.py

```diff
@@ -893,7 +893,7 @@ def __init__(self, controller: Any, msg: Message) -> None:
                     urwid.Text(field),
                     (max_widths[1], urwid.Text(data))
                 ], dividechars=2),
-                None if index % 2 else 'bar')
+                None if index % 2 else 'popup_contrast')
```

Comment:
:+1: 

We should document this change in the commit description.

Reply:
I thought this was fairly obvious, but I've added a short description.